include/tscore/Extendible.h

```diff
@@ -98,9 +98,9 @@ template <typename T> class has_super_type
 // C API
 
 // context (internally FieldDesc*)
-typedef void const *ExtFieldContext;
-typedef void *DerivedPtr;
-typedef void *FieldPtr;
+using ExtFieldContext = const void *;
```

Comment:
Note: `void const` is equivalent to `const void`

Reply:
True. And I generally prefer `something const` over `const somethin`, but this is what VSCode auto fixed to. Most of this PR is an application of autofix from VS Code. If you want I'll change it, but the the more the autofix did over me manually doing it, the safer I feel. :)

Reply:
I'm +1 with staying auto-fixed code.

I was saying to myself that this change is good because I am confused sometimes with the combination of const and pointer :)